src/debuginfo.cpp

```diff
@@ -774,9 +775,11 @@ static int lookup_pointer(DIContext *context, jl_frame_t **frames,
 
     int fromC = (*frames)[0].fromC;
     int n_frames = inlineInfo.getNumberOfFrames();
-    if (n_frames == 0)
+    if (n_frames == 0) {
+        JL_UNLOCK(&codegen_lock);
```

Comment:
Well, it segfaults right now, so much much worse would that be ;)? Is there something else we can do?